docs/doc/60-contributing/03-rfcs/20220721-new-logging.md

```diff
@@ -0,0 +1,155 @@
+---
+title: The New Logging
+description: Make Databend Logging Great Again!
+---
+
+- RFC PR: [datafuselabs/databend#6729](https://github.com/datafuselabs/databend/pull/6729)
+- Tracking Issue: [datafuselabs/databend#0000](https://github.com/datafuselabs/databend/issues/0000)
+
+# Summary
+
+Introduce a new configuration to make the databend's logging more user-friendly, and create more space for further improvement.
+
+# Motivation
+
+Databend's logging is verbose.
+
+Every log will be written to both files and the console. Users can't control the logging behavior.
+
+For examples:
+
+- Users can't specify the `JSON` / `TEXT` format for stderr or files.
+- Users can't disable the file's logging
+
+Even worse, we can't leave meaningful messages to users. Our users will be overwhelmed by a large number of logs. We should use `stdout` to communicate with the user.
+
+# Guide-level explanation
+
+With this RFC, users will have new config options:
+
+```toml
+[log.file]
```

Comment:
despite the application logs, we may have seperated log types, currently there's also query logs, it's also seperated logged into different locations

Reply:
CockroachDB has an idea of logging channels. Maybe it is what you want?

https://www.cockroachlabs.com/docs/v22.1/logging.html